docs/design/autoware-interfaces/ad-api/types/autoware_ad_api_msgs/srv/change_operation_mode.md

```diff
@@ -0,0 +1,19 @@
+# autoware_ad_api_msgs/srv/ChangeOperationMode
+
+## Definition
+
+```txt
+uint16 DIRECT = 1
```

Comment:
Not a strong opinion, I think we can use `NONE` here because it means "Autoware leaves from controlling the vehicle".

@yukkysaito @mitsudome-r @TakaHoribe What do you think?

```suggestion
uint16 NONE = 0
```